frontend/src/utils/formatHelpers.js

```diff
@@ -1,18 +1,23 @@
 function URLParamToString(param) {
-  if (param === 'middle' || param === 4) {
+  // param's possible values:
+  // - In our BE we store it as "Middle" or "Intermediate"
+  // - In our app router we route as "/middle" or "/intermediate"
+  if (param.toLowerCase && param.toLowerCase() === 'middle') {
     return 'Middle School';
   }
-  if (param === 'intermediate' || param === 8) {
-    return 'Intermediate Level';
+  if (param.toLowerCase && param.toLowerCase() === 'intermediate') {
```

Comment:
should it be `toLowerCase()` or was that the intended thing?

Reply:
I think this is checking for the existence of the toLowerCase method before trying to run it. To defend against `undefined is not a function` or something like that. But Xuan Li please correct me if im wrong

Reply:
Aye, that was the purpose. It's also cause I was in a state of having to support both the "legacy" version of `param` being a number and the "new" version of `param` being a string.

Anyway, now that we've defined `consts`, I used them for comparison instead so that future changes can be done just by changing the defined const. ⚡️